Orth_tree: Incorrect documentation

Doing some extra tests on the CGAL documentation (setting `GENERATE_XML=YES`), the following warning appeared:
```
.../Orthtree/include/CGAL/Orthtree.h:860: warning: argument 'search_point' of command @param is not found in the argument list of CGAL::Orthtree< Traits_, PointRange_, PointMap_ >::nearest_k_neighbors_in_radius(Sphere &query_sphere, std::size_t k, OutputIterator output) const
.../Orthtree/include/CGAL/Orthtree.h:860: warning: argument 'search_radius_squared' of command @param is not found in the argument list of CGAL::Orthtree< Traits_, PointRange_, PointMap_ >::nearest_k_neighbors_in_radius(Sphere &query_sphere, std::size_t k, OutputIterator output) const
.../Orthtree/include/CGAL/Orthtree.h:860: warning: The following parameter of CGAL::Orthtree::nearest_k_neighbors_in_radius(Sphere &query_sphere, std::size_t k, OutputIterator output) const is not documented:
  parameter 'query_sphere'
```
Even though the function is not part of the official documentation the documentation of the function should be correct.

--- a/AABB_tree/include/CGAL/AABB_tree.h
+++ b/AABB_tree/include/CGAL/AABB_tree.h
@@ -563,10 +563,10 @@ public:
     /**
      * @brief Builds the tree by recursive expansion.
      * @param first the first primitive to insert
-     * @param last the last primitive to insert
+     * @param beyond the last primitive to insert
      * @param range the number of primitive of the range
      *
-     * [first,last[ is the range of primitives to be added to the tree.
+     * [first,beyond[ is the range of primitives to be added to the tree.
      */
     template<typename ConstPrimitiveIterator, typename ComputeBbox, typename SplitPrimitives>
     void expand(Node& node,

--- a/Orthtree/include/CGAL/Orthtree.h
+++ b/Orthtree/include/CGAL/Orthtree.h
@@ -855,19 +855,19 @@ private: // functions :
   }
 
   /*!
-    \brief finds the `k` points within a specific radius that are nearest to `query`.
+    \brief finds the `k` points within a specific radius that are
+    nearest to the center of `query_sphere`.
 
     This function guarantees that there are no closer points than the ones returned,
     but it does not guarantee that it will return at least `k` points.
     For a query where the search radius encloses `k` or fewer points, all enclosed points will be returned.
-    If the search radius passed is too small, no points may be returned.
+    If the search radius is too small, no points may be returned.
     This function is useful when the user already knows how sparse the points are,
     or if they do not care about points that are too far away.
     Setting a small radius may have performance benefits.
 
     \tparam OutputIterator must be a model of `OutputIterator` that accepts points
-    \param search_point the location to find points near
-    \param search_radius_squared the size of the region to search within
+    \param query_sphere the region to search within
     \param k the number of points to find
     \param output the output iterator to add the found points to (in order of increasing distance)
    */

--- a/Poisson_surface_reconstruction_3/include/CGAL/Poisson_reconstruction_function.h
+++ b/Poisson_surface_reconstruction_3/include/CGAL/Poisson_reconstruction_function.h
@@ -763,8 +763,7 @@ private:
   /// Poisson reconstruction.
   /// Returns false on error.
   ///
-  /// @commentheading Template parameters:
-  /// @param SparseLinearAlgebraTraits_d Symmetric definite positive sparse linear solver.
+  /// @tparam SparseLinearAlgebraTraits_d Symmetric definite positive sparse linear solver.
   template <class SparseLinearAlgebraTraits_d>
   bool solve_poisson(
     SparseLinearAlgebraTraits_d solver, ///< sparse linear solver
@@ -1203,8 +1202,7 @@ private:
 
   /// Assemble vi's row of the linear system A*X=B
   ///
-  /// @commentheading Template parameters:
-  /// @param SparseLinearAlgebraTraits_d Symmetric definite positive sparse linear solver.
+  /// @tparam SparseLinearAlgebraTraits_d Symmetric definite positive sparse linear solver.
   template <class SparseLinearAlgebraTraits_d>
   void assemble_poisson_row(typename SparseLinearAlgebraTraits_d::Matrix& A,
                             Vertex_handle vi,

--- a/Poisson_surface_reconstruction_3/include/CGAL/Reconstruction_triangulation_3.h
+++ b/Poisson_surface_reconstruction_3/include/CGAL/Reconstruction_triangulation_3.h
@@ -348,12 +348,12 @@ public:
   /// Insert the [first, beyond) range of points in the triangulation using a spatial sort.
   /// Default type is INPUT.
   ///
-  /// @commentheading Template Parameters:
-  /// @param InputIterator iterator over input points.
-  /// @param PointPMap is a model of `ReadablePropertyMap` with a value_type = Point_3.
+  /// *Template Parameters:*
+  /// @tparam InputIterator iterator over input points.
+  /// @tparam PointPMap is a model of `ReadablePropertyMap` with a value_type = Point_3.
   ///        It can be omitted if InputIterator value_type is convertible to Point_3.
-  /// @param NormalPMap is a model of `ReadablePropertyMap` with a value_type = Vector_3.
-  ///
+  /// @tparam NormalPMap is a model of `ReadablePropertyMap` with a value_type = Vector_3.
+  /// @tparam Visitor the visitor type
   /// @return the number of inserted points.
 
   // This variant requires all parameters.

--- a/Poisson_surface_reconstruction_3/include/CGAL/poisson_refine_triangulation.h
+++ b/Poisson_surface_reconstruction_3/include/CGAL/poisson_refine_triangulation.h
@@ -180,7 +180,7 @@ public:
 /// bad means badly shaped or too big).
 /// @return the number of vertices inserted.
 ///
-/// *Preconditions:*
+/// \pre
 /// - Tr must use a geometric traits with robust circumcenter computation.
 /// - convergence is guaranteed if radius_edge_ratio_bound >= 1.0.
 ///

--- a/Poisson_surface_reconstruction_3/include/CGAL/poisson_refine_triangulation.h
+++ b/Poisson_surface_reconstruction_3/include/CGAL/poisson_refine_triangulation.h
@@ -180,17 +180,17 @@ public:
 /// bad means badly shaped or too big).
 /// @return the number of vertices inserted.
 ///
-/// @commentheading Preconditions:
+/// *Preconditions:*
 /// - Tr must use a geometric traits with robust circumcenter computation.
 /// - convergence is guaranteed if radius_edge_ratio_bound >= 1.0.
 ///
-/// @commentheading Template Parameters:
-/// @param Tr 3D Delaunay triangulation.
-/// @param Surface Sphere_3 or Iso_cuboid_3.
-/// @param Sizing_field A sizing field functor type
-/// @param Second_sizing_field A sizing field functor type
+/// *Template Parameters*
+/// @tparam Tr 3D Delaunay triangulation.
+/// @tparam Surface Sphere_3 or Iso_cuboid_3.
+/// @tparam Sizing_field A sizing field functor type
+/// @tparam Second_sizing_field A sizing field functor type
 ///
-/// @commentheading Sizing fields
+/// *Sizing fields*
 /// - The first sizing field is the real sizing field that is targeted by
 /// the refinement process. It may be costly to use.
 /// - The second sizing field is supposed to be a sizing field that is less

--- a/Polyhedron/include/CGAL/Polyhedron_3.h
+++ b/Polyhedron/include/CGAL/Polyhedron_3.h
@@ -1448,10 +1448,10 @@ public:
 
     /// Erases the small connected components and the isolated vertices.
     ///
-    /// @commentheading Preconditions:
+    /// *Preconditions:*
     /// supports vertices, halfedges, and removal operation.
     ///
-    /// @commentheading Template Parameters:
+    /// *Parameters*
     /// @param nb_components_to_keep the number of large connected components to keep.
     ///
     /// @return the number of connected components erased (ignoring isolated vertices).

--- a/Polyhedron/include/CGAL/Polyhedron_3.h
+++ b/Polyhedron/include/CGAL/Polyhedron_3.h
@@ -1448,8 +1448,8 @@ public:
 
     /// Erases the small connected components and the isolated vertices.
     ///
-    /// *Preconditions:*
-    /// supports vertices, halfedges, and removal operation.
+    ///
+    /// \pre supports vertices, halfedges, and removal operation.
     ///
     /// *Parameters*
     /// @param nb_components_to_keep the number of large connected components to keep.

--- a/Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h
+++ b/Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h
@@ -1209,7 +1209,8 @@ private:
    * @param t concurrency tag
    */
   template<typename ProxyWrapperIterator>
-  void fit(const ProxyWrapperIterator beg, const ProxyWrapperIterator end, const CGAL::Sequential_tag &) {
+  void fit(const ProxyWrapperIterator beg, const ProxyWrapperIterator end, const CGAL::Sequential_tag & t) {
+    CGAL_USE(t);
     std::vector<std::list<face_descriptor> > px_faces(m_proxies.size());
     for(face_descriptor f : faces(*m_ptm))
       px_faces[get(m_fproxy_map, f)].push_back(f);
@@ -1230,7 +1231,8 @@ private:
    * @param t concurrency tag
    */
   template<typename ProxyWrapperIterator>
-  void fit(const ProxyWrapperIterator beg, const ProxyWrapperIterator end, const CGAL::Parallel_tag &) {
+  void fit(const ProxyWrapperIterator beg, const ProxyWrapperIterator end, const CGAL::Parallel_tag & t) {
+    CGAL_USE(t);
     std::vector<std::list<face_descriptor> > px_faces(m_proxies.size());
     for(face_descriptor f : faces(*m_ptm))
       px_faces[get(m_fproxy_map, f)].push_back(f);
@@ -1334,7 +1336,7 @@ private:
    * 3. Update the proxy error.
    * 4. Update proxy map.
    * @pre current face proxy map is valid
-   * @param face_descriptor face
+   * @param f face
    * @param px_idx proxy index
    * @return fitted wrapped proxy
    */
@@ -1790,7 +1792,7 @@ private:
   /*!
    * @brief walks along the region boundary cycle to the first halfedge
    * pointing to a vertex associated with an anchor.
-   * @param[in/out] he_start region boundary halfedge
+   * @param[in,out] he_start region boundary halfedge
    */
   void walk_to_first_anchor(halfedge_descriptor &he_start) {
     const halfedge_descriptor start_mark = he_start;
@@ -1805,7 +1807,7 @@ private:
   /*!
    * @brief walks along the region boundary cycle to the next anchor
    * and records the path as a `Boundary_chord`.
-   * @param[in/out] he_start starting region boundary halfedge
+   * @param[in,out] he_start starting region boundary halfedge
    * pointing to a vertex associated with an anchor
    * @param[out] chord recorded path chord
    */
@@ -1818,7 +1820,7 @@ private:
 
   /*!
    * @brief walks to the next boundary cycle halfedge.
-   * @param[in/out] he_start region boundary halfedge
+   * @param[in,out] he_start region boundary halfedge
    */
   void walk_to_next_border_halfedge(halfedge_descriptor &he_start) const {
     const std::size_t px_idx = get(m_fproxy_map, face(he_start, *m_ptm));

--- a/Surface_mesher/include/CGAL/IO/output_surface_facets_to_polyhedron.h
+++ b/Surface_mesher/include/CGAL/IO/output_surface_facets_to_polyhedron.h
@@ -23,17 +23,17 @@
 
 namespace CGAL {
 
-/// \deprecated Gets reconstructed surface out of a SurfaceMeshComplex_2InTriangulation_3 object.
+/// \deprecated Gets reconstructed surface out of a `SurfaceMeshComplex_2InTriangulation_3` object.
 ///
 /// This variant exports the surface as a polyhedron.
 /// It requires the surface to be manifold. For this purpose,
-/// you may call make_surface_mesh() with Manifold_tag or Manifold_with_boundary_tag parameter.
+/// you may call `make_surface_mesh()` with `Manifold_tag` or `Manifold_with_boundary_tag` parameter.
 ///
-/// @commentheading Template Parameters:
-/// @param SurfaceMeshComplex_2InTriangulation_3 model of the SurfaceMeshComplex_2InTriangulation_3 concept.
-/// @param Polyhedron an instance of CGAL::Polyhedron_3<Traits>.
+/// *Template Parameters:*
+/// @tparam SurfaceMeshComplex_2InTriangulation_3 model of the `SurfaceMeshComplex_2InTriangulation_3` concept.
+/// @tparam Polyhedron an instance of `CGAL::Polyhedron_3<Traits>`.
 ///
-/// @return true if the surface is manifold and orientable.
+/// @return `true` if the surface is manifold and orientable.
 template <class SurfaceMeshComplex_2InTriangulation_3,
           class Polyhedron>
 CGAL_DEPRECATED_MSG(

